src/@types/auth.ts

```diff
@@ -90,3 +90,19 @@ export enum SSOAction {
     /** The user intends to register for a new account */
     REGISTER = "register",
 }
+
+/**
+ * The result of a successful [MSC3882](https://github.com/matrix-org/matrix-spec-proposals/pull/3882)
+ * `m.login.token` issuance request.
+ * Note that this is UNSTABLE and subject to breaking changes without notice.
+ */
+export interface ILoginTokenPostResponse {
```

Comment:
We should not prefix interfaces with `I` according to our new [code style](https://github.com/vector-im/element-web/blob/develop/code_style.md)